cluster/examples/kubernetes/ceph/nfs.yaml

```diff
@@ -41,3 +41,5 @@ spec:
     #  requests:
     #    cpu: "500m"
     #    memory: "1024Mi"
+    # the priority class to set to influence the scheduler's pod preemption
+    priorityClassName:
```

Comment:
Is this un-commented for a reason? All the rest seem to be commented out.

Reply:
I had it commented out at first but it ends up looking like its part of the comments. 
I suppose I could also add a line that says something along the lines of "uncomment below".

An empty string for the PriorityClassName is the default in Kubernetes so figured it didn't hurt to have it uncommented.

I am open to either approach.

Reply:
If it doesn't matter, I think it's probably better to get this PR in. :+1: